pkgs/applications/networking/cluster/k8sgpt/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "k8sgpt";
+  version = "0.2.9";
+
+  src = fetchFromGitHub {
+    owner = "k8sgpt-ai";
+    repo = "k8sgpt";
+    rev = "v${version}";
+    hash = "sha256:039yiwyh84l1dm6b4vzabv427kfpxj41bjs53cqq85xmqah34bdm";
+  };
+
+  vendorHash = "sha256-nutpoc8UAPXgvBr4hUTYq9/d5OQXGOl3QNtaWhH9Cpw=";
+
+  CGO_ENABLED = 0;
```

Comment:
Any particular reason why you chose to disable `CGO_ENABLED`? :)

In nixpkgs, CGO is usually kept enabled.
See https://github.com/NixOS/nixpkgs/pull/224819#issuecomment-1500890578

Reply:
Mostly it's about what the upstream project is following. I don't have any preference for CGO unless it's required.

Reply:
I think I would prefer it if you could remove `CGO_ENABLED = 0;` so we can keep the nixpkgs default :)

But no worries if you don't want to.

Not blocking^^

Reply:
I would use nixpkgs default approach, unless that breaks the program itself. Meaning not to disable CGO_ENABLED.

Reply:
> I would use nixpkgs default approach, unless that breaks the program itself. Meaning not to disable CGO_ENABLED.

@developer-guy, Your thoughts on this?